app/controllers/items_controller.rb

```diff
@@ -5,18 +5,34 @@ class ItemsController < ApplicationController
   before_action :find_item, only: %i[show toggle_favorite]
 
   def index
-    # Filter by postal code
-    if params[:postal_code].present?
+    # Gets current user coordinates, currently using static postal code until we fix the Js issue.
+    @current_postal_code = 'H2T 1X3'
+
+    # Distance between current user and other users (for index cards).
+    @distances_between_other_users = {}
+
+    # Sets distance for each user that's nearby, private method below.
```

Comment:
Private method defined below, it calculates the distance between the current use and all other users.